wordpress-mobile/WordPress-iOS
# Show post title in "Customize the message" screen if no publicized message.

Fixes #10665 

To test:
1. Go to blog posts
2. Go to Drafts
3. Tap on a draft post
4. tap on the 3 dots icon in the top right corner
5. Go to Post Settings
6. Under Sharing section tap on the message
7. notice that the message appears in the next screen and is editable
8. edit message and go back.
9. Notice that the message was updated.
10. Tap on message again, clear text and go back
11. Notice that the message is now back to the ortiginal post title. 

![publicizedmessage](https://user-images.githubusercontent.com/1335657/51090270-76842780-1747-11e9-9082-c386412acdfb.gif)

![simulator screen shot - iphone 8 - 2019-01-15 at 12 50 23](https://user-images.githubusercontent.com/1335657/51209067-538b7c00-18dd-11e9-8ffe-9d239d06e2ba.png)


Update release notes:
* Fixed an issue where the message field in "Customize the message" screen was empty.
* Added clarification text to footer about empty messages.
